http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -49,13 +51,15 @@ private FeignClientFactory(
             Decoder decoder,
             ErrorDecoder errorDecoder,
             Function<Optional<SSLSocketFactory>, Client> clientSupplier,
-            BackoffStrategy backoffStrategy) {
+            BackoffStrategy backoffStrategy,
+            Request.Options options) {
```

Comment:
should we support a default constructor which does not require providing this parameter to allow compatibility?